adagrams/game.py

```diff
@@ -1,13 +1,82 @@
-from random import randint
+from random import randint 
 
+# define letter pool
+LETTER_POOL = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 
+    'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 
+    'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+}
+# define score chart
+SCORE_CHART = {
+    1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'], 
+    2: ['D', 'G'], 
+    3: ['B', 'C', 'M', 'P'],
+    4: ['F', 'H', 'V', 'W', 'Y'],
+    5: ['K'], 
+    8: ['J', 'X'],
+    10: ['Q', 'Z']
+}
+
+# make letter pool of all potential letters
+def get_all_letters():
+    full_letter_list = []
+    for letter, frequency in LETTER_POOL.items():
+        full_letter_list += [letter] * frequency
+    return full_letter_list
+
+# generate 10 random letters using random index of full letter list 
 def draw_letters():
-    pass
+    full_letter_list = get_all_letters()
+    hand = []
+    for i in range(10):
```

Comment:
If we aren't using the `i` in our loop then convention is to use `_` instead.
```suggestion
    for _ in range(10):
```